dvc/command/config.py

```diff
@@ -85,13 +101,24 @@ def add_parser(subparsers, parent_parser):
         help=CONFIG_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    config_parser.add_argument(
+    config_parser.set_defaults(func=CmdConfig)
+
+    edit_group = config_parser.add_argument_group("edit")
+    edit_group.add_argument(
         "-u",
         "--unset",
         default=False,
         action="store_true",
         help="Unset option.",
     )
-    config_parser.add_argument("name", help="Option name.")
-    config_parser.add_argument("value", nargs="?", help="Option value.")
-    config_parser.set_defaults(func=CmdConfig)
+    edit_group.add_argument("name", nargs="?", help="Option name.")
+    edit_group.add_argument("value", nargs="?", help="Option value.")
```

Comment:
`dvc config section.option` will print the value for that option, so I guess we shouldn't put it into an `edit` group.

Reply:
Agreed! We can either move it to the parent parser (which seems a bit odd, since it is not actually required for `--list`) or change the name of this group (to something like `update`, which I'd assume semantically means `when both <value> and <unset> is not passed, just return the value without changing it. Else, do the required action`).

Reply:
@isidentical Alternative might be to group them by action/type similar to how `git config` does it, but that would be too much work and will make it inconsistent with the rest of our CLI. Maybe let's just not introduce groups at all for now? If I understand correctly, right now groups only have a cosmetical function, not really affecting the logic in the code.